xchange-kucoin/src/main/java/org/knowm/xchange/kucoin/KucoinTradeServiceRaw.java

```diff
@@ -43,7 +59,7 @@ protected KucoinTradeServiceRaw(KucoinExchange exchange) {
                 nonceFactory,
                 passphrase,
                 symbol,
-                null,
+                orderId,
```

Comment:
what was this behavior before adding order id?

Reply:
OrderId is optional and supported by kucoin api. Since there is no common api to get fills for single order we are calling getKucoinFills directly. Well similar to DTO where all properties are listed but only some are mapped to xchange common api.